.github/workflows/build-test-distribute.yaml

```diff
@@ -121,6 +127,12 @@ jobs:
     timeout-minutes: 10
     if: ${{ always() }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      actions: read # For getting workflow run info
+    env:
+      SECURITY_ASSETS_DOWNLOAD_PATH: "${{ github.workspace }}/security-assets"
+      SECURITY_ASSETS_PACKAGE_NAME: "${{ needs.check.outputs.SECURITY_ASSETS_PACKAGE_NAME }}"
```

Comment:
Why is this configured through mk? Can this just be `security-assets` ?

Reply:
done